org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/smoketest/internal/AbstractScenarioRunner.java

```diff
@@ -59,6 +59,7 @@ protected void runChild(final FrameworkMethod method, RunNotifier notifier) {
 		}
 	}
 
+	@Override
```

Comment:
what about `org.eclipse.xtext.testing.smoketest.internal.AbstractScenarioRunner`

Reply:
There is no warning in that class.

Reply:
is this a tooling/settings error?

Reply:
junit4 has project specific settings testing has not imgo it was forgotten to crate/checkin

Reply:
Yes, that's what sure missing there.